reinforcement_learning/include/ranking_response.h

```diff
@@ -34,40 +34,40 @@ namespace reinforcement_learning {
     /**
      * @brief Construct a new ranking response object.  
      * 
-     * @param uuid Unique identifier for this interaction.  The same uuid must also be presented with report_outcome()
+     * @param event_id Unique identifier for this interaction.  The same event_id must also be presented with report_outcome()
      */
-    ranking_response(char const* uuid);
+    ranking_response(char const* event_id);
     /**
      * @brief Unique id for this ranking request.  
      * The same unique id must be passed back when reporing outcome so it can be
      * tied back to the chosen action.
      * @return const char* 
      */
-    const char* get_uuid() const;
+    const char* get_event_id() const;
 
     /**
-     * @brief Get the choosen action id.  
+     * @brief Get the chosen action id.  
      * 
      * @param action_id Chosen action id
      * @param status Optional field with detailed string description if there is an error 
      * @return int Error code
      */
-    int get_choosen_action_id(size_t& action_id, api_status* status = nullptr) const; // id of the top action chosen by the ds
+    int get_chosen_action_id(size_t& action_id, api_status* status = nullptr) const; // id of the top action chosen by the ds
```

Comment:
I am not a huge fan of the comment `by the ds`.
May be better to remove `by the ds`, or replace with `by the decision service`?

Reply:
✔️